aspnetcore/tutorials/signalr.md

```diff
@@ -155,7 +155,6 @@ The output looks like the following example:
 
 ```console
 wwwroot/js/signalr/dist/browser/signalr.js written to disk
```

Comment:
And maybe will be better to add the exactly what you got when run the command 
```
Downloading file https://unpkg.com/@microsoft/signalr@latest/dist/browser/signalr.js...
wwwroot/js/signalr/dist/browser/signalr.js written to disk
Installed library "@microsoft/signalr@latest" to "wwwroot/js/signalr"
```

Reply:
Hello @GeeSuth ... Is that complete? Does it actually have an ellipsis ("`...`") at the end of the "Downloading" line, or did you not show the full line?

Reply:
I'll go ahead with an ellipsis there. I think you mean that the line is quite a bit longer.

Reply:
@guardrex sorry for delay, it's actually ended with ellipsis 

![image](https://github.com/dotnet/AspNetCore.Docs/assets/10328974/e7fb8512-9bdf-4868-b011-492faf06840d)


Thank you for merge!